### PR TITLE
Sanita_Submitting_Module_1

### PR DESCRIPTION
**Pull request by szayshlyy**

### CODE REVIEW

**Comment by szayshlyy** [comment]
Submitting module 1.

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+        "handle-callback-err": "error",
+        "id-blacklist": "error",
+        "id-length": [
+            "off",
+            {
+                "properties": "never"
+            }
+        ],
+        "id-match": "error",
+        "indent": "off",
```
Comment (jamdon) [inline .eslintrc.js]: When you repeat rules, the last one present wins. This one turns off the indent rule you set up near the top (it's set to "off" when you want "error".)

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+        "no-sync": "error",
+        "no-ternary": "off",
+        "no-throw-literal": "error",
+        "no-trailing-spaces": "error",
+        "no-undef-init": "error",
+        "no-undefined": "error",
+        "no-underscore-dangle": "error",
+        "no-unmodified-loop-condition": "error",
+        "no-unneeded-ternary": "error",
+        "no-unused-expressions": "error",
```
Comment (jamdon) [inline .eslintrc.js:155]: This is missing the no-unused-vars rule.

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+        "complexity": "error",
+        "computed-property-spacing": "error",
+        "consistent-return": "error",
+        "consistent-this": "error",
+        "curly": "error",
+        "default-case": "error",
+        "dot-location": "error",
+        "dot-notation": "error",
+        "eol-last": "error",
+        "eqeqeq": "off",
```
Comment (jamdon) [inline .eslintrc.js]: This rule should be active, not off. Use "error" instead.

**Comment by jamdon** [comment]
You don't have any linting errors, but you should in this case as you don't have all the rules active you need for the assignment. Additionally, you are missing a rule, but it appears you've fixed the linting issues it would have reported anyway. You should still add the rule (no-unused-vars) in regardless. This is very close - let me know if you want it graded as-is or make updates and let me know if you want it re-reviewed.

**Comment by szayshlyy** [comment]
Hi James, thanks for the feedback! 
I will try to re-work this assignment and turn it back in hopefully by the end of the day.

**Comment by jamdon** [comment]
Looks good. 👍

**Pull request closed without merging**